### PR TITLE
repo: Use `jq` toolchain

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message:
Additional Description:

afaict this will work on linux/mac(x64)/win:

https://github.com/aspect-build/bazel-lib/blob/main/lib/private/jq_toolchain.bzl

Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
cc @keith

File: bazel/repository_locations.bzl [hunk 1]
```diff
@@ -100,6 +100,18 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2019-08-08",
         cpe = "cpe:2.3:a:google:boringssl:*",
     ),
+    aspect_bazel_lib = dict(
+        project_name = "Aspect Bazel helpers",
+        project_desc = "Base Starlark libraries and basic Bazel rules which are useful for constructing rulesets and BUILD files",
+        project_url = "https://github.com/aspect-build/bazel-lib",
+        version = "0.11.10",
```
Comment (moderation) [inline bazel/repository_locations.bzl:107]: `0.12.0` is out - https://github.com/aspect-build/bazel-lib/releases/tag/v0.12.0

Comment (phlax) [reply bazel/repository_locations.bzl:107]: the wheel keeps turning!

**Review by keith — APPROVED** [review]

**Comment by wrowe** [comment]
This looks clean, makes things easier for docs authors to test locally

/lgtm deps

**Pull request merged**